### ISSUE TITLE
How to change user Name and password 

### ISSUE THREAD

**Issue by sandeep-g-more** [K-G-PRAJWAL/ReactJS-Projects#23]
How can i change the user name and password
So that i can practice by making changes  in the user name and password

**Comment by K-G-PRAJWAL** [comment]
Which project are you talking about?

**Comment by VinayChaurasiyaA** [comment]
yeah @sandeepganapatimore  can you specify which project you talking about, so that we can help you!

**Comment by VinayChaurasiyaA** [comment]
@K-G-PRAJWAL Hey i didn't find the issue but can I as an individual add my project in here?
